### ISSUE TITLE
Bad Smell Code can be see in Sonar Web but don't report to gitlab

### ISSUE THREAD

**Issue by ChanShuYi** [gabrie-allaigre/sonar-gitlab-plugin#33]
So sorry to interrupt you again. @gabrie-allaigre 

I add some code with some bad smell, and then I use following command to analysis the code,but the sonar don't report issue to gitlab. 

```
mvn --batch-mode verify sonar:sonar -Dsonar.analysis.mode=preview -Dsonar.gitlab.commit_sha=5cbd43da4a1e81ac59e27e96cb0ad6c0a1abe5ed -Dsonar.gitlab.ref_name=master -Dsonar.gitlab.user_token=xgxpcGw-TzsRDmkFG-9S -Dsonar.gitlab.project_id=git@gitlab.chenyurong.com:chenyurong/zhenai-commons.git 
```

the command execute result

```
[INFO] -------------  Scan zhenai-commons
[INFO] Load server rules
[INFO] Load server rules (done) | time=178ms
[INFO] Base dir: /home/chenyurong/Code/dubbo/zhenai-commons
[INFO] Working dir: /home/chenyurong/Code/dubbo/zhenai-commons/target/sonar
[INFO] Source paths: pom.xml, src/main/java
[INFO] Test paths: src/test/java
[INFO] Binary dirs: target/classes
[INFO] Source encoding: UTF-8, default locale: en_US
[INFO] Index files
[INFO] 128 files indexed
[INFO] Quality profile for java: Sonar way
[INFO] Sensor XmlFileSensor
[INFO] Sensor XmlFileSensor (done) | time=0ms
[INFO] Load server issues
[INFO] Load server issues (done) | time=1666ms
[INFO] Performing issue tracking
[INFO] 164/164 components tracked
[INFO] ANALYSIS SUCCESSFUL
[INFO] Executing post-job GitLab Commit Issue Publisher
[INFO] Report status=success, desc=SonarQube reported no issues
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 31.133 s
[INFO] Finished at: 2017-04-25T03:49:56-07:00
[INFO] Final Memory: 27M/102M
[INFO] ------------------------------------------------------------------------
```

gitlab can receive reporting from sonar but no issue which can be see in Sonar Web as the screenshot below.

![](http://images2015.cnblogs.com/blog/595137/201704/595137-20170425185806444-1806785297.png)

**Comment by gabrie-allaigre** [comment]
Plugin report only new bug, if bug is write in SonarQube Web site then no reporting per plugin.

**Comment by ChanShuYi** [comment]
you are so amazing, thanks your help.
